publication-rest/src/test/java/no/unit/nva/publication/validation/PublicationUriValidatorTest.java

```diff
@@ -0,0 +1,22 @@
+package no.unit.nva.publication.validation;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import java.net.URI;
+import org.junit.jupiter.api.Test;
+
+public class PublicationUriValidatorTest {
```

Comment:
Probably, the person that wrote PublicationUriValidator didn't think about having the same host, which they should have done.